terraform/environments/delius-jitbit/s3.tf

```diff
@@ -56,3 +56,46 @@ resource "aws_s3_bucket_intelligent_tiering_configuration" "jitbit_bucket_tierin
     days        = 180
   }
 }
+
+
+module "datasync_bucket" {
```

Comment:
Is there something here we need to specify to enforce bucket ownership over the objects?